services/src/datasets/external/aruna/mod.rs

```diff
@@ -650,11 +650,10 @@ impl ArunaDataProvider {
             cache_ttl,
         };
 
-        GdalLoadingInfo {
-            info: GdalLoadingInfoTemporalSliceIterator::Static {
-                parts: vec![part].into_iter(),
-            },
-        }
+        // FIXME: someone with more detailed insights needs to add the time bounds!
```

Comment:
maybe present the problem to Niki on Wednesday?

Reply:
yes but cant reference a commit here

Reply:
if we ask Niki we can insert a better comment here, of even fix it and then merge the PR

Reply:
[2a0012f](https://github.com/geo-engine/geoengine/pull/968/commits/2a0012f1e7a20e0ecd34dbf85995cb4163403080)